tests/gold_tests/basic/deny0.test.py

```diff
@@ -33,16 +33,11 @@
 
 ts = Test.MakeATSProcess("ts", enable_cache=False)
 ts.Disk.records_config.update({
-     'proxy.config.diags.debug.enabled': 1
-    ,'proxy.config.diags.debug.tags': 'http|dns|redirect'
-    ,'proxy.config.http.redirection_enabled': 1
-    ,'proxy.config.http.number_of_redirections': 1
-    ,'proxy.config.dns.nameservers': '127.0.0.1:{0}'.format(dns.Variables.Port)
-    ,'proxy.config.dns.resolv_conf': 'NULL'
-    ,'proxy.config.url_remap.remap_required': 0  # need this so the domain gets a chance to be evaluated through DNS
+    # need this so the domain gets a chance to be evaluated through DNS
+    'proxy.config.diags.debug.enabled': 1, 'proxy.config.diags.debug.tags': 'http|dns|redirect', 'proxy.config.http.redirection_enabled': 1, 'proxy.config.http.number_of_redirections': 1, 'proxy.config.dns.nameservers': '127.0.0.1:{0}'.format(dns.Variables.Port), 'proxy.config.dns.resolv_conf': 'NULL', 'proxy.config.url_remap.remap_required': 0
```

Comment:
Can we keep these separated into multiple lines?

Reply:
Good catch.

Interesting. It looks like the original formatting confused autopep8. I updated it, now running it produces a better result.

Reply:
OK, it looks like the initial ', ' format was throwing off autopep8. I fixed things here and in doc/conf.py.

Thanks for noticing this.